internal/v2/remote_signer/remote_signer.go

```diff
@@ -0,0 +1,12 @@
+// Remote signer library that offloads private key operations to S2Av2.
+package remotesigner
+
+import (
+	"crypto"
+	"crypto/tls"
+)
+
+// NewRemoteSigner returns an instance of RemoteSigner, an implemenation of the crypto.Signer interface.
+func NewRemoteSigner(cert *tls.Certificate) crypto.Signer {
+	return cert.PrivateKey.(crypto.Signer)
```

Comment:
Shall we add a TODO here for the actual implementation? This also tells a reader that this is not the actual implementation :)